lib/constants.js

```diff
@@ -37,7 +37,8 @@ module.exports = Object.freeze({
   },
   HTTP_STATUS_CODE: {
     CONFLICT: 409,
-    UNPROCESSABLE_ENTITY: 422
+    UNPROCESSABLE_ENTITY: 422,
+    BAD_REQUEST: '400'
```

Comment:
is it okay we are putting the 400 status code as string, while others are number ?

Reply:
Changed to int.